ml_enabler/models/ml_model.py

```diff
@@ -10,7 +12,9 @@ class Prediction(db.Model):
     created = db.Column(db.DateTime, default=timestamp, nullable=False)
     model_id = db.Column(db.BigInteger, db.ForeignKey(
                         'ml_models.id', name='fk_models'), nullable=False)
-    predictions = db.Column(db.JSON, nullable=False)
+    version = db.Column(db.Integer, nullable=False)
+    bbox = db.Column(Geometry('MULTIPOLYGON', srid=4326))
```

Comment:
@geohacker does this need to be `MULTIPOLYGON` - is there a simpler `POLYGON` or `BBOX` type that maybe better?

Reply:
There does not seem to be a `BBOX` type: https://geoalchemy-2.readthedocs.io/en/0.3/types.html, but perhaps `POLYGON` is better than `MULTIPOLYGON` ?

Reply:
I thought about it and it might make sense to allow multipolygon in case the model bboxes are funny. But I'll wait for @wronk to weigh in.

Reply:
@geohacker, any efficiency cost to going with `multipolygon`? If no, I think we should use it. There may be cases where we're not analyzing tiled imagery (e.g., COGs) and want to store more interesting prediction shapes.

Reply:
I think for simplicity, we should stick to POLYGON. The model prediction shape doesn't necessarily have to be the same as this polygon. The module that pushes prediction data can decide what's the best way to break up into bboxes. 

Let's chat about this more. For now, let us go ahead with POLYGON.